From: Ilya Dryomov <idryomov@redhat.com>
Date: Thu, 13 Nov 2014 20:42:35 +0300
Subject: [PATCH] ceph: remove unused stringification macros
Git-commit: ca3995ad13b5e8bb08be850d08fe1e2abffc0206
Patch-mainline: v3.19
References: Fate#318586


These were used to report git versions a long time ago.

Signed-off-by: Ilya Dryomov <idryomov@redhat.com>
Acked-by: NeilBrown <neilb@suse.com>

---
 fs/ceph/super.c |    3 ---
 1 file changed, 3 deletions(-)

--- a/fs/ceph/super.c
+++ b/fs/ceph/super.c
@@ -1017,9 +1017,6 @@ static struct file_system_type ceph_fs_t
 };
 MODULE_ALIAS_FS("ceph");
 
-#define _STRINGIFY(x) #x
-#define STRINGIFY(x) _STRINGIFY(x)
-
 static int __init init_ceph(void)
 {
 	int ret = init_caches();
